api/src/main/java/net/md_5/bungee/util/Supplier.java

```diff
@@ -0,0 +1,5 @@
+package net.md_5.bungee.util;
+
+public interface Supplier<T> {
```

Comment:
I should add that this is here because BungeeCord is Java 7 and the functional interfaces weren't introduced until Java 8, so it is likely not available on most J7 platforms